### PR TITLE
Set experimental property in ui if url parameter passed

### PR DESCRIPTION
**Pull request by alexpelan**

### CODE REVIEW

**Comment by alexpelan** [comment]
Travis is gonna fail because of the cyclomatic complexity of the ui reducer, let me know if you have any feedback

File: src/actions/applicationLoaded.js [hunk 1]
```diff
@@ -2,5 +2,5 @@ import {createAction} from 'redux-actions';
 
 export default createAction(
   'APPLICATION_LOADED',
-  (gistId = null) => ({gistId}),
+  (gistId = null, isExperimental = false) => ({gistId, isExperimental}),
```
Comment (outoftime) [inline src/actions/applicationLoaded.js]: IMO the action creator should probably just take a named args object at this point (probably should have even before this change)

**Comment by alexpelan** [comment]
@outoftime is that what you meant? Also bumped up complexity to 30

**Comment by outoftime** [comment]
Yep! @alexpelan gonna merge now but would you mind writing up a brief PR description (what the motivation for this is etc.) at your leisure?

**Pull request merged**